ts/spi/provider_frame.ts

```diff
@@ -344,6 +354,22 @@ export class ProviderFrame {
     return this.handleUnimplementedRequest(id, msg.RPC_MESSAGE_TYPES.proxy);
   }
 
+  private async handleCancelLastOperation(id: string) {
+    if (!this.requestInProgress || this.cancellable === null) {
+      // no request in progress
+      console.warn('No pending request to cancel.');
+      this.clientChannel.send(msg.errorMessage(
+          id, OpenYoloError.requestFailed('No request in progress.')));
+    } else {
+      try {
+        this.cancellable.cancel();
+      } finally {
+        this.dispose();
```

Comment:
Yes, that is the only way to dispose of any pending UI right now. @TMSCH has been talking about adding a method on interaction provider which dismisses (and animates) all UI away.